src/PowerShellEditorServices/Services/PowerShellContext/Session/PSReadLinePromptContext.cs

```diff
@@ -95,9 +83,24 @@ internal class PSReadLinePromptContext : IPromptContext
                 if (psReadLineType == null)
                 {
                     logger.LogWarning("PSConsoleReadline type not found: {Reason}", pwsh.HadErrors ? pwsh.Streams.Error[0].ToString() : "<Unknown reason>");
-                    return false;
+                    System.Console.WriteLine("psReadLineType is null, searching loaded assemblies...");                    
+
+                    var allAssemblies = AppDomain.CurrentDomain.GetAssemblies();
+                    var assemblies = allAssemblies.FirstOrDefault(a => a.FullName.Contains("Microsoft.PowerShell.PSReadLine2"));
+                    var type = assemblies?.ExportedTypes?.FirstOrDefault(a => a.FullName == "Microsoft.PowerShell.PSConsoleReadLine");
+                    if(type is not null)
+                    {
+                        System.Console.WriteLine("Found PSConsoleReadLine in loaded assemblies.");                    
```

Comment:
@andschwa I didn't think this was going to be necessary, but upon inspecting the logs from the most recent build, it appears it is:
https://dev.azure.com/powershell/PowerShellEditorServices/_build/results?buildId=81345&view=logs&j=5aed7330-0c52-5e0f-84aa-a17a2c59b4c4&t=c5143455-bea5-57f2-b80c-f2c098a4d98c&l=928
```
Attempting to import D:\a\1\s\module\PSReadLine
psReadLineType is null, searching loaded assemblies...
Found PSConsoleReadLine in loaded assemblies.
```